commcare_export/minilinq.py

```diff
@@ -459,8 +475,8 @@ class Emit(MiniLinq):
     def __init__(
         self,
         table: str,
-        headings: ListType[str],
-        source: ListType[MiniLinq],
+        headings: ListType[MiniLinq],
+        source: MiniLinq,
         missing_value=None,
         data_types=None,
```

Comment:
should these 2 also have hints?

Reply:
https://github.com/dimagi/commcare-export/pull/213